website/docs/r/service_access.html.markdown

```diff
@@ -26,11 +33,15 @@ resource "cf_service_access" "org1-mysql-512mb" {
 The following arguments are supported:
 
 * `plan` - (Required) The ID of the service plan to grant access to
-* `org` - (Required) The ID of the Org which should have access to the plan
+* `org` - (Optional) The ID of the Org which should have access to the plan. Conflicts with `public`.
+* `public` - (Optional) Boolean that controls the public state of the plan. Conflicts with `org`.
 
 ## Import
 
-The current Service Access can be imported using the `service_access`, e.g.
+The current Service Access can be imported using an `id`.
+
+If given `id` matches existing [`service_plan_visibilities`](https://apidocs.cloudfoundry.org/280/service_plan_visibilities/list_all_service_plan_visibilities.html),
+resource will be imported as a `service_access` targeting an organization. Otherwise, it will be imported as `service_access` controlling plan's public state. E.g.
```

Comment:
@psycofdj 

I'd suggest the following clarification to doc:

```
If the given `id` matches [a service plan id](http://apidocs.cloudfoundry.org/280/service_plans/updating_a_service_plan.html), then the resource will be imported as `service_access` controlling plan's public state.

Otherwise, the import would fail
```